### ISSUE TITLE
Apparent recursion within a with_prototype action: 25000 context sanity limit hit

### ISSUE THREAD

**Issue by pinkhominid** [JeremyBernier/LitElement-Syntax-Highlighting#2]
First off, thanks for building this package.

After installing, latest Sublime (v3.2.2 Build 3211) throws this error.

>Error loading syntax file "Packages/Markdown/Markdown.sublime-syntax": Apparent recursion within a with_prototype action: 25000 context sanity limit hit

It appears there is some conflict that occurs with certain plugins. [Issue #2444](https://github.com/sublimehq/sublime_text/issues/2444) over at sublime_text has some discussion about it.

I am running other plugins in tandem and have yet to confirm this plugin solo with base sublime replicates the issue. I'll try later today.

**Comment by JeremyBernier** [comment]
Yes this plugin somehow triggers that annoying Sublime bug. I'm going to update the README.md right now. Hopefully we can find a solution.

**Comment by pinkhominid** [comment]
Cool thanks! Like you, I've partially switched. With some tweaks Atom is alright, but Sublime still feels best to me. So fast and gets out of the way.
